### ISSUE TITLE
Issue on page /installation.html

### ISSUE THREAD

**Issue by vinothm14** [ray-project/ray#15700]
I was using ray in Windows10 machine until  yesterday. Ray helped me a lot, but i accidentally uninstall the python. now i couldnot able to install the ray again.

I am using ANACONDA 3 with Python 3.8.5
I am getting the error @ "Collecting gpustat"
I have attached the error message as well.
![2021-05-08 18_38_03-jEdit - Untitled-1 (modified)](https://user-images.githubusercontent.com/33394807/117547416-cc272100-b02f-11eb-8084-0afcb930ea7b.jpg)

**Comment by simon-mo** [comment]
You can use `pip install --no-dependencies ray` to get around with this error for now. We are in the process of removing/hiding `gpustat` as a dependency.

**Comment by mattip** [comment]
I think this is fixed on the 1.9.1 release from last week:
```
> pip install ray
...
Installing collected packages: wrapt, pyparsing, six, pyrsistent,
packaging, deprecated, colorama, attrs, redis, pyyaml, protobuf, numpy,
msgpack, jsonschema, grpcio, filelock, click, ray
Successfully installed ...

> pip install ray[serve]
...
Successfully built gpustat nvidia-ml-py3
Installing collected packages: pyasn1, sniffio, rsa, pyasn1-modules, 
multidict, frozenlist, cachetools, ansicon, yarl, wcwidth, 
typing-extensions, jinxed, googleapis-common-protos, google-auth, 
async-timeout, anyio, aiosignal, starlette, pydantic, psutil, 
opencensus-context, nvidia-ml-py3, hiredis, h11, google-api-core, 
blessed, asgiref, aiohttp, uvicorn, smart-open, py-spy, prometheus-client,
opencensus, gpustat, fastapi, colorful, aioredis, aiohttp-cors
```

Note that gpustat is successfully installed.

**Comment by mattip** [comment]
Closing, please reopen if I misunderstood.
